### PR TITLE
Feature/js charts

### PR DESCRIPTION
**Pull request by trasher**
This is based on #1454 which needs to be merged first.

### CODE REVIEW

File: inc/html.class.php [hunk 1]
```diff
@@ -5801,6 +5805,10 @@ static public function requireJs($name) {
          case 'jcrop':
             $_SESSION['glpi_js_toload'][$name][] = 'jqueryplugins/jcrop/jquery.Jcrop.min.js';
             break;
+         case 'charts':
+            $_SESSION['glpi_js_toload']['charts'][] = 'chartist-js-0.10.1/chartist.js';
+            $_SESSION['glpi_js_toload']['charts'][] = 'chartist-plugin-legend-0.6.0/chartist-plugin-legend.js';
+            $_SESSION['glpi_js_toload']['charts'][] = 'chartist-plugin-tooltip-0.0.17/chartist-plugin-tooltip.min.js';
```
Comment (orthagh) [inline inc/html.class.php]: miss break ?

File: inc/html.class.php [hunk 2]
```diff
@@ -1180,6 +1180,10 @@ static function includeHeader($title='', $sector = 'none', $item = 'none', $opti
          echo Html::css($CFG_GLPI["root_doc"]."/lib/jqueryplugins/jquery-ui-scrollable-tabs/css/jquery.scrollabletab.min.css");
       }
 
+      echo Html::css($CFG_GLPI["root_doc"]."/lib/chartist-js-0.10.1/chartist.min.css");
```
Comment (orthagh) [inline inc/html.class.php]: shouldn't we need to condition it with 
```php
if (in_array('charts', $jslibs)) {
...
}
```

**Comment by orthagh** [comment]
In addition, i still have some overlap between legend and tooltip.

peek 05-01-2017 15-56

**Comment by trasher** [comment]
OK, there was a z-index issue with tooltips and legend. All should be OK now.

**Comment by orthagh** [comment]
just a last comment, the animation delay is a little long, i wait for the end of animation.

I think about 500ms instead 1s could be better

**Comment by orthagh** [comment]
oh, another subject, tooltips in pie charts don't display any labels.
We need to match legend and it's not very easy to read

**Comment by trasher** [comment]
I don't know if this is possible to get required data from tooltips; cannot find in the doc.

**Comment by trasher** [comment]
All changes have been done

**Comment by orthagh** [comment]
```
PHP Notice: Undefined variable: i in ./inc/stat.class.php at line 1456
```

**Comment by trasher** [comment]
Ooops... fixed.

**Pull request merged**